include/Core/BulkDelete.php

```diff
@@ -467,6 +472,22 @@ private function get_delete_metas_admin_page() {
 		return $metas_page;
 	}
 
+	/**
+	 * Get Bulk Delete Terms admin page.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @return \BulkWP\BulkDelete\Core\Terms\DeleteTermsPage
+	 */
+	private function get_delete_terms_admin_page() {
```

Comment:
Private functions should use `@access private` tag in the Docblock as per [WordPress inline documentation](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/) standards.

Reply:
@mariadanieldeepak, @sudar says ignore the change